### ISSUE TITLE
Disambiguate `earned` from `earned_income`

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/openfisca-us#1070]
`earned_income = employment_income + self_employment_income`
Whereas
`earned = earned_income - self_employment_tax_adjustment`

Can we name these more precisely and verify we're using them in the right places?

**Comment by nikhilwoodruff** [comment]
Yes, we should move `earned` to `tc_earned`. I'd favour renaming to `labour_income` too, because it seems more precise.

**Comment by MaxGhenis** [comment]
If we're using the variable, we should give it a normal name, which we then alias to taxcalc names like `earned`. "Earned income" seems to be the standard description for this variable, in both tax and benefit programs, so I'd suggest we keep it.

**Comment by DTrim99** [comment]
@MaxGhenis Closing as resolved/obsolete — the standalone earned variable was removed; only earned_income remains, so the naming ambiguity is gone (no recent activity). Reopen if it recurs.
